packages/ketcher-core/src/domain/entities/DrawingEntitiesManager.ts

```diff
@@ -166,6 +166,16 @@ export class DrawingEntitiesManager {
     return this.allEntities.length !== 0;
   }
 
+  public get hasMonomers() {
+    const monomers = [...this.monomers.values()].filter(
```

Comment:
`filter` method doesn't mutate the original array and creates a new copy, perhaps you can just use `this.monomers.values().filter()` here?

Reply:
this.monomers.values() returns Iterator, so destructuring operator needs to create array from Iterator.